src/content/docs/quick-installation-new-operator.md

```diff
@@ -86,3 +145,12 @@ The Open Data Hub operator is available for deployment in the OpenShift Operator
 
 - to use "modelmesh" component, users are required to install one operator via OperatorHub before enable it in DataScienceCluster CR
    1. Prometheus Operator from "Community" catalog.
+
+### Limitation
+
+We offer a feature that allows users to configure the namespace for their application. By default, the ODH operator utilizes the `opendatahub` namespace. However, users have the flexibility to opt for a different namespace of their choice. This can be accomplished by modifying the `DSCInitialization` instance with the `.spec.applicationsNamespace` field.
+
+There are two scenarios in which this feature can be utilized:
+
+- Assigning a New Namespace: Users can set a new namespace immediately after creating the `DSCInitialization` instance, before creating the `DataScienceCluster` instance.
```

Comment:
This can be updated to `Users can set a new namespace using `DSCInitialization` instance, before creating the `DataScienceCluster` instance`.

This is since users are not creating `DSCInitialization` instance.